EMAILC-54: IMAP not applying filtering on server side results in larg… (#1093)
mulesoft/docs-connectors: email/1.3/modules/ROOT/pages/email-studio-configuration.adoc

[[create-mule-project]]
== Create a Mule Project

In Studio, create a new Mule project in which to add and configure the connector:

. In Studio, select *File > New > Mule Project*.
. Enter a name for your Mule project and click *Finish*.

[[add-connector-to-project]]
== Add the Connector to Your Mule Project

Add Email Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:

. In the *Mule Palette* view, click *(X) Search in Exchange*.
. In the *Add Dependencies to Project* window, type `email` in the search field.
. Click *Email Connector* in *Available modules*.
. Click *Add*.
. Click *Finish*.

Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.

[[configure-input-source]]
== Configure a Source

A source initiates a flow when a specified condition is met.

> You can configure one of these input sources to use with Database Connector:

Reviewer: ```suggestion
You can configure one of these input sources to use with Email Connector:
```

* *On New Email - IMAP* +
Initiates a flow by retrieving all the emails from an IMAP mailbox folder, watermark can be enabled for polled items
* *On New Email - POP3* +
Initiates a flow by retrieving all the emails from an POP3 mailbox folder
* *HTTP > Listener* +
Initiates a flow each time it receives a request on the configured host and port
* *Scheduler* +
Initiates a flow when a time-based condition is met

> For example, to configure *On New Email - IMAP* source , follow these steps:

Reviewer: ```suggestion
For example, to configure the *On New Email - IMAP* source , follow these steps:
```

| SMTP (Simple Mail Transfer Protocol) a| * *Send*
|===

For example, to configure a POP3 global element for the *List POP3* operation, follow these steps:

. Select the name of the connector in the Studio canvas.
. Select the *List POP3* operation in the Studio canvas.
. In the *List POP3* configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
On the *Email POP3* window, for *Connection* select any of the connection types to provide to this configuration:

* *POP3 Connection*
* *POP3S Connection*

[start=6]
. On the *General* tab, specify the connection information for the connector, such as *Host*, *Port* and *TLS configuration*.
. On the *Advanced* tab, optionally specify timeout configuration and reconnection information, including a reconnection strategy.
. Click *OK*.

The following screenshot shows the Email Connector Global Element Configuration window in Anypoint Studio:

.Email Connector Global Element Configuration
image::email-global-configuration.png[Email Connector Global Element Configuration]

[[configure-other-fields]]
== Configure Additional Connector Fields

> For IMAPS, POP3S, and SMPTPs protocol connections, you can use Transport Layer Security (TLS) and configure email by providing a key store with your certificate. Additionally, you can also enable 2-way authentication by providing a trust store. For details, see xref:email-documentation.adoc#Tls[Email Connector Reference] and xref:mule-runtime::tls-configuration.adoc[Configure TLS with Keystores and Truststores].

Reviewer: ```suggestion
For IMAPS, POP3S, and SMPTPs protocol connections, you can use Transport Layer Security (TLS) and configure email by providing a keystore with your certificate. You can also enable two-way authentication by providing a truststore. For details, see xref:email-documentation.adoc#Tls[Email Connector Reference] and xref:mule-runtime::tls-configuration.adoc[Configure TLS with Keystores and Truststores].
```
